Watchman.Discord/Program.cs

```diff
@@ -18,28 +21,45 @@ public static async Task Main(string[] args)
             var configPath = "config-prod.json";
 #endif
             var configuration = JsonConvert.DeserializeObject<DiscordConfiguration>(File.ReadAllText(configPath));
-            
+
             var watchman = new WatchmanBot(configuration);
             var workflowBuilder = watchman.GetWorkflowBuilder();
-            var lastException = DateTime.MinValue;
-            var numberOfExceptionsInLastTime = 0;
 
-            while (numberOfExceptionsInLastTime++ < HOW_MANY_EXCEPTIONS_IN_SHORT_TIME_TO_STOP_BOT)
+            AppDomain.CurrentDomain.UnhandledException += (sender, e) =>
+            {
+                HandleException((Exception)e.ExceptionObject);
+            };
+            while (true)
+            {
```

Comment:
Jak bot nie działa w takim stopniu że ma problemy z uruchomieniem sie, to nie ma sensu żeby próbował zrobić to samo wiele razy
Lepiej dawać w takich przypadkach Task.Delay(1000) albo 2000
A do tego pętle for, powiedzmy z 5 próbami